api/routes/events/events.ts

```diff
@@ -71,34 +93,31 @@ router.post(
     >,
   ) => {
     try {
-      const id = uuidV4();
-      await pg.query(sql`
-              INSERT INTO events (
-                  id,
-                  name,
-                  description,
-                  start_date,
-                  end_date,
-                  adress
-              ) VALUES (
-                  ${id},
-                  ${req.body.name},
-                  ${req.body.description},
-                  ${req.body.start_date},
-                  ${req.body.end_date},
-                  ${req.body.adress}
-              );
-          `);
-      await pg.query(sql`
-        INSERT INTO events_admin (
-            event,
-            admin
-        ) VALUES (
-            ${id},
-            ${req.userId}
+      const { name, description, startDate, endDate, adress, genres } =
+        req.body;
+
+      const admin = new Musician();
+      admin.id = req.userId;
+
+      const newEvent = new Event();
+      newEvent.name = name;
+      newEvent.description = description;
+      newEvent.startDate = startDate;
+      newEvent.endDate = endDate;
+      newEvent.adress = adress;
+      newEvent.admins = [admin];
```

Comment:
```suggestion
      const newEvent = new Event({name, description, startDate, endDate, adress, admin});
```
Can't you just do something like that?

Reply:
We can if we create constructor for each entity, i will do that

Reply:
Done without create constructors but instead just calling the `getRepository(T).create<T>()`